projects/js-packages/publicize-components/src/components/form/index.js

```diff
@@ -53,7 +53,12 @@ export default function PublicizeForm( {
 		isSocialImageGeneratorEnabled && isSocialImageGeneratorEnabledForPost;
 	const Wrapper = isPublicizeDisabledBySitePlan ? Disabled : Fragment;
 
-	const brokenConnections = connections.filter( connection => false === connection.is_healthy );
+	const brokenConnections = connections.filter(
+		connection => false === connection.is_healthy && 'broken' === connection.error_code
```

Comment:
I wonder if we should check for `connection.error_code` being `undefined` here. Until the corresponding change is deployed on WPCOM then you don't get any notices for broken connections because of this change, so something like that could mean we don't need to worry about the order things are deployed.